Source/AtomicEditor/Editors/ResourceEditor.cpp

```diff
@@ -65,6 +65,7 @@ class EditorTabLayout: public TBLayout
                 }
                 else
                 {
+                    editor_->SendEvent("EditorResourceCloseCanceled");
```

Comment:
In native code, we need to define events like this: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicEditor/Editors/SceneEditor3D/SceneEditor3DEvents.h

... and not use String -> StringHash promotion, on the script side, defining a const var for event name, or using string itself is ok.  The binding generator needs a way to dump all the event interface defs (this would mean modifying the MACRO to capture the event info, which is non-trivial)

There could be a ResourceEditorEvents.h created for resource editor specific events